lib/zigler/zig.ex

```diff
@@ -30,19 +30,19 @@ defmodule Zigler.Zig do
     src_file = Path.basename(compiler.code_file)
     cmd_opts = ["build-lib", src_file] ++
       ~w(-dynamic -lc) ++ cross_compile(compiler.module_spec) ++
-      ~w(--disable-gen-h --override-lib-dir) ++
+      ~w(--override-lib-dir) ++
       [zig_rpath] ++
       include_opts ++
-      ["--ver-major", "#{version.major}",
-       "--ver-minor", "#{version.minor}",
-       "--ver-patch", "#{version.patch}"] ++
+      ["--version", "#{version}"] ++
       lib_opts ++
       ["--name", "#{module}"] ++
-      ["--release-safe"]
+      ~w(-O ReleaseSafe)
```

Comment:
maybe we could pass this in from the end project's `config.exs` ?

Reply:
fyi in 0.7.0 branch I've totally flipped over to using build.zig